db/schema.rb

```diff
@@ -47,13 +47,12 @@
   end
 
   create_table "benefit_receipts", force: :cascade do |t|
-    t.uuid "assessment_id", null: false
+    t.uuid "assessment_id"
```

Comment:
`db/schema.rb` seems to contain changes unrelated to this PR.
I suggest undoing this change and then running `bin/rails db:schema:load` to align your local DB